client/src/components/info/events.js

```diff
@@ -0,0 +1,23 @@
+const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July',
```

Comment:
Why those constant are placed in info folder? Why this folder is placed in components? How developers can connect words components, info and React component Events?

Reply:
As those constants aren't React Components you shouldn't change previous behavior (refactoring). Or if you absolutely sure that those constants are used in different places (right now, not in future) you can move those in separate files in folders like `shaded/constants`.
If you sure that those constants are used only in one component (Search component) you shouldn't move them from this component.

The same message concerns other constants, which are used only in one places, but you moved those in separate files

Reply:
On front part of our app, we show these categories as property values of our entities from back part (moths, event categories, etc). So, I guess we should get it from back part. BUT: due to my poor experience in this question, I just wanted to move these constants up. 

I agree that this in not best decision, but I moved 'hard-code' up, and, finally, some day [may be, I hope] we will get it from back. I want to believe that our code should be less changable and independent. 

Due to our requirements we will need this values in profile history (i.e. search my events). BUT: again, we will get this from back part also. So, if we had no requirements for it, I would probably do the same.

That's why our components shoud render with same code even with different incoming data. It means that we should move 'modifile' level up and up, not inside developer. I like your logic and opinion. I just wanted to show that component should be fully independent.

And yes, we can move it to folder up and name it constants, it is good idea.